### ISSUE TITLE
last_update_date property might be missing

### ISSUE THREAD

**Issue by 1ucian0** [Qiskit/qiskit-ibmq-provider#673]
Move from https://github.com/Qiskit/qiskit-terra/issues/4460

### Information

- **Qiskit IBMQ Provider version**: 0.7.1
- **API URL or version**:

### What is the current behavior?

[This](https://travis-ci.com/github/Qiskit/qiskit-terra/jobs/334547024) execution failed with the following error.

```
Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):
      File "/home/travis/build/Qiskit/qiskit-terra/qiskit/test/decorators.py", line 180, in _wrapper
    return func(self, *args, **kwargs)
      File "/home/travis/build/Qiskit/qiskit-terra/test/python/tools/monitor/test_backend_monitor.py", line 36, in test_backend_overview
    backend_overview()
      File "/home/travis/build/Qiskit/qiskit-terra/qiskit/tools/monitor/backend_overview.py", line 188, in backend_overview
    props = _backends[count].properties().to_dict()
      File "/home/travis/virtualenv/python3.6.10/lib/python3.6/site-packages/qiskit/providers/ibmq/ibmqbackend.py", line 268, in properties
    decode_backend_properties(api_properties)
      File "/home/travis/virtualenv/python3.6.10/lib/python3.6/site-packages/qiskit/providers/ibmq/utils/json_decoder.py", line 57, in decode_backend_properties
    properties['last_update_date'] = dateutil.parser.isoparse(properties['last_update_date'])
    KeyError: 'last_update_date'
```

### What is the expected behavior?

I'm not sure if `last_update_date` is optional. If so, no error. Otherwise, a nice exception.

### Suggested solutions

Assuming it's optional, `qiskit/providers/ibmq/utils/json_decoder.py` should do something like this
```
properties['last_update_date'] = dateutil.parser.isoparse(properties['last_update_date']) if properties.get('last_update_date') else None
```

**Comment by 1ucian0** [comment]
Other instances of, what it looks like, the same error:
 - https://github.com/Qiskit/qiskit-terra/issues/4459 https://travis-ci.com/Qiskit/qiskit-terra/jobs/334547025

**Comment by jyu00** [comment]
@1ucian0 `last_update_date` is a required field: https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/schemas/backend_properties_schema.json#L107

The builds are failing in terra because it uses a user ID that has access to internal testing backends, which do not have proper calibration files. It's fixed by Qiskit/qiskit-terra#4296 hence only fails on the stable branch. I'd recommend disabling online tests for the stable branch or use a different user ID.
